downstream/modules/platform/proc-completing-post-installation-tasks.adoc

```diff
@@ -3,75 +3,17 @@
-
-To ensure that automation controller downloads the execution environment from automation hub, you must import the hub’s Certificate Authority (CA) certificate as a trusted certificate on the controller. You can do this in one of two ways, depending on whether SSH is available as root user between {ControllerName} and {PrivateHubName}. 
-
-=== Using secure copy (SCP) as a root user
-
-If SSH is available as the root user between the controller and {PrivateHubName}, use SCP to copy the root certificate on the {PrivateHubName} to the controller.
-
-
-.Procedure
-
- . Run `update-ca-trust` on the controller to update the CA trust store:
-
-----
-$ sudo -i
-# scp <hub_fqdn>:/etc/pulp/certs/root.crt
-/etc/pki/ca-trust/source/anchors/automationhub-root.crt
-# update-ca-trust
-----
-
-=== Copying and pasting as a non root user
-
-If SSH is unavailable as root between the {PrivateHubName} and the controller, copy the contents of the file __/etc/pulp/certs/root.crt__ on the {PrivateHubName} and paste it into a new file on the controller called __/etc/pki/ca-trust/source/anchors/automationhub-root.crt__. 
-
-.Procedure
-
-. Run `update-ca-trust` to update the CA trust store with the new certificate. On the {PrivateHubName}, run:
+After you have completed the installation of {PlatformNameShort}, ensure that {HubName} and {ControllerName} deploy properly.
 
-----
-$ sudo -i
-# cat /etc/pulp/certs/root.crt
-(copy the contents of the file, including the lines with 'BEGIN CERTIFICATE' and
-'END CERTIFICATE')
-----
+Before your first login, you must add your subscription information to the platform. To obtain your subscription information in uploadable form, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/red_hat_ansible_automation_platform_operations_guide/assembly-aap-obtain-manifest-files#assembly-aap-obtain-manifest-files[Obtaining a manifest file] in the Access management and authentication guide.
 
-. On the {ControllerName}:
+Once you have obtained your subscription manifest, see link:{URLGettingStarted}[{TitleGettingStarted}]for instructions on how to upload your subscription information.
 
-----
-$ sudo -i
-# vi /etc/pki/ca-trust/source/anchors/automationhub-root.crt
-(paste the contents of the root.crt file from the private automation hub into the new file and write to disk)
-# update-ca-trust
-----
+Now that you have successfully installed Ansible Automation Platform, to begin using its features, see the following guides for your next steps:
 
+link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html/getting_started/with/Ansible_automation_platform[{TitleGettingStarted}]
 
-.Additional Resources
+link:{LinkHubManagingContent}[Managing content in automation hub]
```

Comment:
No need for the [Managing content in automation hub] it's included in the attribute.

Reply:
Change made.